docs/event.md

```diff
@@ -0,0 +1,93 @@
+
+If present the properties will be decorated and/or tokenised. 
+
+To date we've discussed :-
+
+- If `user.session.token` is included it will be validated against the session api and expanded in to a membership uuid.
+- If `content.uuid` is included it will be expanded in to a set of content api meta-data.
+- ...
+
+## Format
+
+```
+{
+    "event": {
+        "source": "email.simple-email-service",  				// Name of the sender's system [1]
+        "type": "opened",  										// The type of event that happened, Eg. 'pageview', 'dwell', 'play' etc.
+		"api-key":	"0f7464b4-3f4d-11e4-984b-00144feabdc0"		// Sender-specific key [6]
+    },
+    "user": {													// Who generate the event
+        "user-agent": "Mozilla ...",
+        "ip": "123.123.123.123",
+        "anonymous-id": "...",									// An anonymous token, supplied by the client [2] 
+        "session": {
+            "token": "asdf324dfag1ds%asdf1A-1sadsadf",  		// FT Session token - expanded in to a uuid and valid flag [3]
+        },
+		"device": {
+			"layout": "S",										// require('o-grid').getCurrentLayout 
+        	"orientation": "portrait"							// require('o-viewport').getOrientation
+		},
+		"geo": {												// [8]
+            "country": "MA",
+            "area_code": "0",
+            "longitude": "-5.000",
+            "latitude": "32.000",
+            "country_name": "Morocco",
+            "continent": "AF"
+		},
+		"ab": {
+			"test-name": "variant"								// [9]
+		},
+		"flags": [ ],											// Flags/Toggles the user has set [10]
+		"isStaff": true											// [9]
+    },
+    "content": {
+        "uuid": "", 				// Content uuid [4] 
+        "url": "...", 	 			// [5] 
+        "referrer": "...", 			// See also [5]
+		"type":	"content"			// Some sort of controlled list  
+    },
+	"meta": {						// [7]
+		"anything": "goes here",
+		"could: [ "be", "an", "array" ],
+		"or" : { "might": { "be": ["an", "object"] }}
+	},
+    "time": {
+         "offset": 234				// For offline stuff, I think
+    }
+}
+```
+
+```
+[1]  The general convention might be something like `product.sub-system.environment` - Eg, 'webapp.render.uat'
+[2]  Or in absence appended by Spoor. 
+[3]  So, the token becomes -> { session: { token: '...', uuid: '...', valid: true }
+[3]  I will scan any Cookies in the header for a FTSession token as well.
+[4]  Content will typically be a page, but could be a uuid for something else like a video. Will be expanded in to a CAPI v2 object -> capi: { "headline", "...", "published": "...", ... }
+[5]  Expanded in to a URL object -> http://nodejs.org/api/url.html
+[6]  I think I might also allow this in a `authorization` header.
+[7]  This is intentionally vague. Allow anyone to store anything. Consumers own responsibility for filtering what they need.
+[8]  Where the event happened. If not provided this is expanded by a call to Maxmind.
+[9]  An enrichment
+[10] Possibly not that useful?
```

Comment:
qatool and __toggler are the same thing I think (cookie based flags) so this makes sense.

>  it would make sense for flags to be an Object for our uses.

Agree, it should be the same as the AB test segments - i.e. `user.flags -> { "flag": "on", "foo": off }`